bc_obps/bc_obps/settings.py

```diff
@@ -35,6 +35,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'registration.apps.RegistrationConfig',
```

Comment:
Change which part to `registration`?

Reply:
forget about my comment 🫣 I think I should update my Django knowledge since things have changed. It used to be just the app name like `registration` in the `INSTALLED_APP` list every time you add a new app to Django project instead of having all `registration.apps.RegistrationConfig`.